.github/workflows/build-deploy.yml

```diff
@@ -31,5 +31,6 @@ jobs:
           host: ${{ secrets.HOST }}
           user: ${{ secrets.USERNAME }}
           key: ${{ secrets.KEY }}
-          post_upload: java -jar http-server.jar &
+          pre_upload: kill -9 $(lsof -t -i:5000)
```

Comment:
I'm glad you found how to do this!

Reply:
Thanks! I'm actually going to update this script bc I realized it causes the deploy to fail if the server isn't currently running... about to change it to a script that only runs if it finds a process on port 5000